server/modules/authentication/azure/authentication.js

```diff
@@ -37,12 +37,16 @@ module.exports = {
         useCookieInsteadOfSession: keyArray.length > 0,
         cookieEncryptionKeys: keyArray
       }, async (req, iss, sub, profile, cb) => {
-        const usrEmail = _.get(profile, '_json.email', null) || _.get(profile, '_json.preferred_username')
+        const emails = _.get(profile, '_json.emails', null);
```

Comment:
Default value should be an empty array, not null, as it will throw on line 43 otherwise

Reply:
Fixed!